README.rst

```diff
@@ -69,6 +69,254 @@ Windows
+
+You can also create and import a dataset in separate steps:
+
+.. code-block:: Python
+
+    from google.cloud import aiplatform
+
+    my_dataset = aiplatform.TextDataset.create(
+        display_name="my-dataset")
+
+    my_dataset.import(
+        gcs_source=['gs://path/to/my/dataset.csv']
+        import_schema_uri=aiplatform.schema.dataset.ioformat.text.multi_label_classification
+    )
+
+To get a previously created Dataset:
+
+.. code-block:: Python
+  
+  dataset = aiplatform.ImageDataset('projects/my-project/location/us-central1/datasets/{DATASET_ID}')
+
+Vertex AI supports a variety of dataset schemas. References to these schemas are available under the
+:code:`aiplatform.schema.dataset` namespace. For more information on the supported dataset schemas please refer to the
+`Preparing data docs`_.
+
+.. _Preparing data docs: https://cloud.google.com/ai-platform-unified/docs/datasets/prepare
+
+Training
+^^^^^^^^
+The Vertex SDK allows you train Custom and AutoML Models.
+
+Custom models can be trained using a custom Python script, custom Python package, or container.
+
+Preparing Your Custom Code
+--------------------------
+Vertex AI custom training enables you to train on AI Platform Datasets and produce AI Platform Models. To do so your
+script must adhere to the following contract:
+
+1. It must read dataset from the environment variables populated by the training service:
+
+.. code-block:: Python
+
+  os.environ['AIP_DATA_FORMAT']  # provides format of data  
+  os.environ['AIP_TRAINING_DATA_URI']  # uri to training split
+  os.environ['AIP_VALIDATION_DATA_URI']  # uri to validation split
+  os.environ['AIP_TEST_DATA_URI']  # uri to test split
+
+Please visit `Using a managed dataset in a custom training application`_ for a detailed overview.
+
+.. _Using a managed dataset in a custom training application: https://cloud.google.com/vertex-ai/docs/training/using-managed-datasets
+
+2. It must write the model artifact to the environment variable populated by the traing service:
+
+.. code-block:: Python 
+
+  os.environ['AIP_MODEL_DIR']
+
+Running Training
+----------------
+
+.. code-block:: Python
+
+  job = aiplatform.CustomTrainingJob(
+      display_name="my-training-job",
+      script_path="training_script.py",
+      container_uri="gcr.io/cloud-aiplatform/training/tf-cpu.2-2:latest",
+      requirements=["gcsfs==0.7.1"],
+      model_serving_container_image_uri="gcr.io/cloud-aiplatform/prediction/tf2-cpu.2-2:latest",
+  )
+
+  model = job.run(my_dataset,
+                  replica_count=1,
+                  machine_type="n1-standard-4",
+                  accelerator_type='NVIDIA_TESLA_K80',
+                  accelerator_count=1)
+
+In the code block above my_dataset is managed dataset created in the `Datasets` section above. `model` is a Managed Vertex AI Model that the be deployed or exported.
+
+
+AutoMLs
+-------
+The Vertex SDK for Python supports AutoML Tabular, Image, Text, Video, and Forecasting.
```

Comment:
Remove initial caps from "Tabular," "Image," "Text," "Video," and "Forecasting"--these aren't proper nouns.